pinc/wordcheck_engine.inc

```diff
@@ -626,12 +620,16 @@ function count_wordcheck_suggestion_events($projectid, $start_time=0)
 
 function load_wordcheck_events($projectid, $start_time=0)
 {
-    $sql = "SELECT * FROM wordcheck_events
-            WHERE projectid='$projectid' AND timestamp>$start_time";
-
-    $eventArray = array();
+    $sql = sprintf("
+        SELECT * FROM wordcheck_events
+        WHERE projectid='%s',
+        AND timestamp>%d",
+        DPDatabase::escape($projectid),
+        $start_time
+    );
 
-    $res = mysqli_query(DPDatabase::get_connection(), $sql);
+    $eventArray = [];
+    $res = DPDatabase::query($sql, FALSE);
```

Comment:
While this is true to the original, the original is wrong and this should raise on error. If the query returns zero rows (which is possible) `$res` will still be not FALSE.